code/controllers/garbage.dm

```diff
@@ -61,6 +66,28 @@ var/soft_dels = 0
 			if(remainingForceDelPerTick <= 0)
 				break
 
+			#ifdef GC_FINDREF
+			to_chat(world, "picnic! searching [locate(D)]")
+			if(istype(D, /atom/movable))
+				var/atom/movable/A = D
+				testing("GC: Searching references for [A] | [A.type]")
+				if(A.loc != null)
+					testing("GC: [A] | [A.type] is located in [A.loc] instead of null")
+				if(A.contents.len)
+					testing("GC: [A] | [A.type] has contents:")
+					for(var/atom/B in A.contents)
+						testing("[B] | [B.type]")
+			var/found = 0
+			for(var/atom/R in world)
+				found += LookForRefs(R, D)
+			for(var/datum/R)
+				found += LookForRefs(R, D)
+			for(var/A in _all_globals)
+				found += LookForListRefs(readglobal(A), D, null, A)
```

Comment:
Been doing some porting and noted two things thus far:
* You're not checking for references in clients as they're not included in the /datum loop (but they can basically be treated as datums as they too have the vars list). Minor but there might be a reference to newplayer there.
* A member in _all_globals isn't necessarily a list, but could be a reference to an instance of something as well.